src/main/java/com/internetitem/logback/elasticsearch/util/Base64.java

```diff
@@ -0,0 +1,103 @@
+package com.internetitem.logback.elasticsearch.util;
+
+import java.io.ByteArrayOutputStream;
+
+/**
+ * @author Sridharan Kuppa
```

Comment:
Sorry, I did not see the comment.  I will correct and submit the pull request one more time.